CMakeLists.txt

```diff
@@ -689,6 +684,11 @@ target_link_libraries(ktx PRIVATE ${ASTC_LIB_NAME})
 # Its a good optimisation anyways and most compilers now support it
 set_property(TARGET ktx PROPERTY INTERPROCEDURAL_OPTIMIZATION_RELEASE True)
 
+# Use a static runtime on MSVC builds (ignored on non-MSVC compilers)
```

Comment:
Is this comment correct? It looks like you are setting the multi-threaded library for the debug configuration.

Reply:
I think so. You mean its only for debug? If I understand Cmake generator expressions correctly. `"MultiThreaded$<$<CONFIG:Debug>:Debug>")` will be `MultiThreaded` for release and `MultiThreadedDebug` for debug build.

Reply:
This has now disappeared because we have added a `CLI=OFF` option to astcencoder.